src/ol/geom/geometry.js

```diff
@@ -130,8 +130,11 @@ ol.geom.Geometry.prototype.getClosestPoint = function(point, opt_closestPoint) {
 
 
 /**
+ * Returns true if this geometry contains the specified coordinate. If the
+ * coordinate is on the boundary of the geometry, returns false.
```

Comment:
I'm guessing that our implementation doesn't really follow the [traditional definition](http://www.vividsolutions.com/jts/javadoc/com/vividsolutions/jts/geom/Geometry.html#contains(com.vividsolutions.jts.geom.Geometry)) of contains for polygons.  Looking at the code, I'd guess that a point on the boundary of an exterior ring would be considered "contained", while a point on the boundary of an interior ring would not be contained.  A more strict implementation would say that a polygon only contains a point if it is in the interior but not touching a ring.